scripts/main.js

```diff
@@ -1,159 +1,170 @@
-  studentsOnDom(students);
-  }
-};
-
-
-/* FILTER BUTTONS */
-  const filterButtons = () => {
-    const domString = `
-    <div class="navigation-btn">
-  <button id="all">All</button>
-  <button id="Gryffindor">Gryffindor</button>
-  <button id="Hufflepuff">Hufflepuff</button>
-  <button id="Ravenclaw">Ravenclaw</button>
-  <button id="Slytherin">Slytherin</button>
+  
+  /******* STUDENTS *******/
+  const studentsOnDom = (array) => {
+    const student = array.filter(taco => taco.darkside === false);
+    let domString = "";
+    for (const item of student) {
+    domString += `
+    <div class="card rcorners" style="width: 18rem;">
+    <img src="${item.image}" class="card-img-top rcorners" alt="...">
+    <div class="card-body" id="studentCard">
+      <h5 class="card-title">${item.name}</h5>
+      <p class="card-text">${item.school}</p>
+      <a href="#" class="btn btn-primary expelBtn" id="expel--${item.id}" >Expel</a>
     </div>
-    `;
-    renderToDom('#filterStudents', domString);
+  </div>`
   };
-
-
-/****** EVENT LISTENERS ******/
-const eventListeners = () => {
-  document.querySelector("#filterStudents").addEventListener("click", (e) => {
-  if (e.target.id === "all") {
-    studentsOnDom(students);
-  } else if (e.target.id === "Gryffindor") {
-      const gryffindor = students.filter(taco => taco.school === "Gryffindor" )
-      studentsOnDom(gryffindor);
-    } else if (e.target.id === "Hufflepuff") {
-      const hufflepuff = students.filter(taco => taco.school === "Hufflepuff" )
-      studentsOnDom(hufflepuff)
-    } else if (e.target.id === "Ravenclaw") {
-      const ravenclaw = students.filter(taco => taco.school === "Ravenclaw" )
-      studentsOnDom(ravenclaw)
-    } else if (e.target.id === "Slytherin") {
-      const slytherin = students.filter(taco => taco.school === "Slytherin" )
-      studentsOnDom(slytherin)
+  renderToDom("#studentContainer", domString)
+  };
+  
+  const voldOnDom = (array) => {
+    let domString = "";
+    const student = array.filter(taco => taco.darkside === false);
+    for (const item of student) {
+    domString += `
+    <div class="card2 rcorners" style="width: 18rem;">
+    <img src="https://rb.gy/mn64l0" class="card-img-top rcorners" alt="...">
+    <div class="card-body" id="studentCard">
+      <h5 class="card-title">${item.name} has joined</h5>
+      <p class="card-text">Voldermort's Army</p>
+    </div>
+  </div>`
+  };
+  renderToDom("#voldContainer", domString)
+  };
+  
+  /* ADD STUDENTS */
+  function addStudent(){ 
+    const input = document.querySelector("#studentInput").value
+    if (input === "") {
+      alert("Please enter your name!");
+    } else {
+    
+    const randomSchool = choices[Math.floor(Math.random()*choices.length)];
+    const newStudent = {
+      id: students.length + 1,
+      image: "",
+      name: input,
+      school: randomSchool,
+      darkside: false
+    };
+    
+    if (randomSchool === "Gryffindor") {
```

Comment:
Another way to do this would make the `choices` variable an array of objects with the name and the image. So once randomSchool is called a object will be selected and you can dot notate on the house and the image.